assets/dev/js/editor/utils/localized-value-store.js

```diff
@@ -0,0 +1,39 @@
+export default class LocalizedValueStore {
+	constructor() {
+		this.store = [];
+	}
+
+	sendKey( event ) {
+		const isLetter = event.keyCode >= 65 && event.keyCode <= 90;
+		const isSpace = 32 === event.keyCode || ' ' === event.originalEvent.data;
+
+		if ( isLetter || isSpace ) {
+			this.store.push( { original: event.originalEvent.key, english: String.fromCharCode( event.keyCode ) } );
+		}
+		if ( ! event.target.value || event.target.value.length < this.store.length ) {
+			this.store = this.rebuildStore( event.target.value );
+		}
+	}
+
+	shouldBeIgnored( event ) {
+		return event.shiftKey || event.ctrlKey || event.altKey;
+	}
+
+	getLocalizedValue() {
```

Comment:
Maybe that's better? Since the consumer knows it's a localized value already (`localizedValue.getLocalizedValue()` is pretty weird) 

```suggestion
	get() {
```